cpp/include/cudf/strings/char_types/char_types.hpp

```diff
@@ -27,7 +27,7 @@ namespace strings
  * @brief Character type values.
  * These types can be or'd to check for any combination of types.
  */
-enum string_character_types {
+enum string_character_types : int32_t {
```

Comment:
We want to make this an `enum class`:
```suggestion
enum class string_character_types : int32_t {
```

Reply:
Connected offline and we use this enum to do `|` and `&` operations which you can't do on enum classes without implementing them, so leaving as an enum for now.